keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -443,107 +456,173 @@ General prep:
 Prep for brownfield:
 + admin creates `Bucket` instances representing backend buckets.
 
+### Finalizers
+The following finalizers are added by COSI to orchestrate deletion of the various COSI resources. Of note is that a finalizer is added to the two user created instances (BR and BAR) so that these intances remain available until the deletion of the associated resources (Bucket and BA) and the backend bucket have completed.
+
++ _objectstorage.k8s.io/user-protect_, added to BRs and BARs.
++ _objectstorage.k8s.io/br-protect_, added to a `Bucket` indicating a BR created it.
++ _objectstorage.k8s.io/ba-\<name1> ba-\<name2>..._, added to a `Bucket` for each BA referencing the `Bucket.`
++ _objectstorage.k8s.io/bar-protect_, added to `BucketAccess` indicating a BAR created it.
++ _objectstorage.k8s.io/pod-\<uuid1> pod-\<uuid2>..._, added to `BucketAccess` for each pod accessing the BA.
```

Comment:
Cfr. comment above.

Reply:
I don't understand, sorry.

Reply:
Was referring to another similar comment: I thought, as discussed in the meetings, we were *not* going to add a finalizer per consuming Pod, but only one, and instead list/watch Pods in the relevant controllers to figure out whether finalization was OK to happen.